modules/rosa-sts-creating-a-cluster-with-customizations.adoc

```diff
@@ -49,7 +49,7 @@ $ rosa create account-roles --mode manual <1>
 +
 .. After review, run the `aws` commands manually to create the roles and policies. Alternatively, you can run the preceding command using `--mode auto` to run the `aws` commands immediately.
 
-. (Optional) If you are using your own AWS KMS key to encrypt the control plane data volumes and the persistent volumes (PVs) for your applications, add the ARN for the account-wide installer role to your KMS key policy.
+. Optional: if you are using your own AWS KMS key to encrypt the control plane data volumes for your applications, add the ARN for the account-wide installer role to your KMS key policy.
```

Comment:
You could remove "for your applications", because control plane data volumes relate to core OpenShift data.